### PR TITLE
Documenting Patient Identifier type

### PR DESCRIPTION
**Pull request by dino-saurabh**
1. added examples and description

### CODE REVIEW

**Review by Akayeshmantha — COMMENTED** [review]
Looks good

**Comment by dino-saurabh** [comment]
> Looks good

just a few more sections to add and then I will make it ready to review @Akayeshmantha

File: source/includes/Patient/patient_identifier_type.md [hunk 1]
```diff
@@ -64,27 +128,30 @@ POST /patientidentifiertype
...
 {
     "name": "Amani Identifier",
     "description": "Medical record number for Amani Health System",
     "format": "\\d{1,10}-\\d",
     "formatDescription": "Up to ten digts followed by a hyphen and another digit",
     "required": false,
+    "*checkDigit":true,
     "validator": "org.openmrs.patient.impl.LuhnIdentifierValidator",
     "locationBehavior": "NOT_USED",
     "uniquenessBehavior": "UNIQUE"
```
Comment (dino-saurabh) [inline source/includes/Patient/patient_identifier_type.md]: @Akayeshmantha I am getting a 500 Internal server error with this payload could you get whats wrong with this Json payload? thanks !!

**Comment by dino-saurabh** [comment]
@Akayeshmantha please review !!

**Comment by Akayeshmantha** [comment]
LGTM

**Pull request merged**